example/parallelLoopPatterns/src/parallelLoopPatterns.cpp

```diff
@@ -177,7 +179,11 @@ void gridStridedLoop(TDev& dev, TQueue& queue, TBufAcc& bufAcc)
     std::cout << "\nGrid strided loop processing - fixed number of threads and blocks:\n";
     std::cout << "   " << blocksPerGrid << " blocks, " << threadsPerBlock << " threads per block, "
               << "alpaka element layer not used\n";
-    alpaka::exec<TAcc>(queue, workDiv, GridStridedLoopKernel{}, std::data(bufAcc), n);
+
+    GridStridedLoopKernel gridStridedLoopKernel;
```

Comment:
please revert this change, there is no need to change this. The old interface should work too.